### ISSUE TITLE
Address breadcrumbs for IA items that are not pages

### ISSUE THREAD

**Issue by amy-super** [grafana/grafana#52276]
In the recent [POC user test](https://docs.google.com/presentation/d/1TKS7ILN30mC6TCBSNnC7efU1vnpQ_Bl3tdTJgjBVq_E/edit#slide=id.g10b7a169e20_0_12), multiple users got hung up using the breadcrumbs when they included an item that represents a group, rather than a page. We need a plan for how to handle this - create overview pages? Remove those grouping from the breadcrumbs? Something else?
<img width="904" alt="Screen Shot 2022-07-15 at 7 53 06 AM" src="https://user-images.githubusercontent.com/87499219/179218060-d83c32a5-06d2-4fb8-9734-1963b4139c6a.png">

**Comment by zsofiakomaromigrafana** [comment]
I totally agree with this approach, and with bringing back Home as well. Even this way, the breadcrumbs will be a great help when you're deeper in the hierarchy and want to move a level or multiple levels up. Since this has been the major challenge that we've been trying to solve.

As for "Home", on Grafana Cloud that would most probably be the Cloud homepage, and perhaps the "Home dashboard" ("Personal dashboard" etc.) could be under dashboards.

**Comment by amy-super** [comment]
OK I've documented the breadcrumb behavior [in Figma](https://www.figma.com/file/jylhhvPsRKdGR5xlmIqXem/Start-here---Navigation?node-id=873%3A3076) so I'm going to close this issue.

**Comment by amy-super** [comment]
We decided to revisit this decision. 

- Because of the groupings in the IA, we run into some real weirdness with leaving conceptual categories out of the breadcrumbs. For example, Synthetic Monitoring’s Settings page would have a breadcrumb trail of Home > Settings.
- It won’t be a huge amount of work to auto-generate landing page
- But we’d have to make a plan for how to handle plugins which don’t necessarily have landing pages - instead they have default pages.

@ashharrison90 has started playing with auto-generating these landing pages [here](https://github.com/grafana/grafana/tree/ash/landing-page-hack).

We also discussed how this might work for places that already have a "home." Next steps - @amy-super to write up a design doc.

**Comment by amy-super** [comment]
[Design doc](https://docs.google.com/document/d/1xcY-rG5eC7CnTq5WVuYaJGOLOurYDVQLDihqFVIrwvA/edit#)

[Figma designs for landing pages](https://www.figma.com/file/jylhhvPsRKdGR5xlmIqXem/Start-here---Navigation?node-id=1330%3A919)
